browser/index.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  extends: ['mixmax', 'plugin:react/recommended'],
+  parser: 'babel-eslint',
+  settings: {
+    react: {
+      version: '15.5.4'
```

Comment:
Oh yeah let's do latest-stable. I think that will be nice to have it tell us when stuff becomes deprecated, it'll be like an update advisory.